tests/admin_filters/tests.py

```diff
@@ -623,6 +627,17 @@ def test_relatedfieldlistfilter_manytomany(self):
         self.assertIs(choice['selected'], True)
         self.assertEqual(choice['query_string'], '?contributors__id__exact=%d' % self.bob.pk)
 
+    def test_relatedfieldlistfilter_reverse_relationships_related_name(self):
+        modeladmin = CustomUserAdminRelatedOnlyRelatedName(User, site)
+
+        request = self.request_factory.get('/')
+        request.user = self.alfred
+
+        try:
```

Comment:
This test should very the correct results of the Filter rather than just checking that an exception isn't raised.